.github/workflows/test-convene-web.yml

```diff
@@ -131,7 +131,7 @@ jobs:
           path: tmp/capybara/*.png
 
   test-features:
-    name: Run browser tests
+    name: Run Cucumber tests
```

Comment:
`Run Cucumber feature tests` seemed like a really long name thus I shortened it. But I can add it back if people prefer it.